### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Emiluren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,13 @@ The game client is very slow in debug mode, so it should be run in release mode
 SDL2 needs to be added to be able to compile Plen under Windows. You can follow this tutorial https://github.com/Rust-SDL2/rust-sdl2/blob/master/README.md#windows-with-build-script
 (or possibly one of the others listed below it, not tested it myself but should work).
 
-You also need to install SDL2 mixer which can be found here https://www.libsdl.org/projects/SDL_mixer/.
+You also need to install SDL2_image, SDL2_mixer and SDL2_ttf which can be found here:
+https://www.libsdl.org/projects/SDL_image/
+https://www.libsdl.org/projects/SDL_mixer/
+https://www.libsdl.org/projects/SDL_ttf/
```
Comment (TheZoq2) [inline README.md]: Make this a list

```
- https://...
- https://...
...
```

File: README.md [hunk 2]
```diff
@@ -19,10 +19,13 @@ The game client is very slow in debug mode, so it should be run in release mode
 SDL2 needs to be added to be able to compile Plen under Windows. You can follow this tutorial https://github.com/Rust-SDL2/rust-sdl2/blob/master/README.md#windows-with-build-script
```
Comment (TheZoq2) [inline README.md]: While you're at it :)

`you can follow [this tutorial](<url>)` for better readability

**Review by TheZoq2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Emiluren) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Emiluren) [reply README.md]: Fixed

**Review by malcx95 — APPROVED** [review]

**Review by TheZoq2 — APPROVED** [review]

**Pull request merged**